content/storyboards.yaml

```diff
@@ -0,0 +1,31 @@
+stories:
+  NW:
+  - title: TBA
+    link: https://eucp-project.github.io/storyboards
+
```

Comment:
Perhaps at some point, we will be able to fetch these stories directly from the storyboards page, e.g.

```js
stories = await fetch("https://eucp-project.github.io/storyboards/_content/stories?region='alpine'&category='cpm'")
```

see https://content.nuxtjs.org/advanced#api-endpoint

Reply:
@Peter9192 Thanks for the tip! We will keep it for further developments later.